application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -902,6 +911,14 @@ type LegTime {
   scheduledTime: OffsetDateTime!
 }
 
+"A level with a name and comparable number. Levels can sometimes contain half levels, e.g. '1.5'."
+type Level {
+  "0-based comparable number where 0 is the ground level."
+  level: Float!
+  "Name if it exists, e.g. 'M' or 'P1'."
+  name: String!
```

Comment:
The value is required but the description says "if it exists"? Will there always be a value?